internal/agent/config.go

```diff
@@ -120,6 +123,7 @@ func NewDefault() *Config {
 		SpecFetchInterval:    DefaultSpecFetchInterval,
 		reader:               fileio.NewReader(),
 		LogLevel:             logrus.InfoLevel.String(),
+		DefaultLabels:        map[string]string{},
```

Comment:
nit: sorry I missed this before it's good practice to instantiate maps with make. Promise last nit :)
```suggestion
		DefaultLabels:        make(map[string]string),
```